src/App.js

```diff
@@ -1,23 +1,23 @@
 import React, { useState } from 'react';
 import { ReactComponent as Arrow } from './arrow.svg'
 import { ReactComponent as Reset } from './reset.svg'
-import Cell from './components/Cell';
+import { shallowEqual, useSelector } from 'react-redux'
 
-function App() {
-
-  const COLORS = ['yellow', 'green', 'blue', 'violet', 'pink', 'red'];
-  const SIZE = 14;
-  
-  function randomNum(max) {
-    return Math.floor(Math.random() * max)
-  }
-
-  const initial = new Array(SIZE)
-  .fill(0)
-  .map(_ => new Array(SIZE).fill(0).map(_ => randomNum(COLORS.length)))
-
-  const [board, changeBoard] = useState(initial);
+function Cell({color}) {
```

Comment:
You will never reuse this particular component. So it make no sense to move it in a separate file